pymemcache/client/base.py

```diff
@@ -233,6 +236,18 @@ def __init__(self,
             store commands (except from cas, incr, and decr, which default to
             False).
           allow_unicode_keys: bool, support unicode (utf8) keys
+          flush_on_reconnect: bool, optional flag which prevents a scenario
```

Comment:
I'd prefer something more like this:

flush_on_reconnect: bool, optional flag which causes a `flush_all()` command to be sent anytime the memcache client is disconnected and then reconnected to a memcached server. If you have a cluster of memcached servers that use a consistent hash ring (or a similar approach to routing) this can prevent stale reads of old keys when a server in the ring is disconnected and then reconnected. Note, however, that a client with an intermittent connection to a memcached server can cause it to be repeatedly flushed, and the same is true for a cluster of memcached servers. That makes this a VERY dangerous thing to enable on a memcache cluster.